deps/libgit2.mk

```diff
@@ -77,10 +77,10 @@ $(LIBGIT2_SRC_PATH)/libgit2-bundled_zlib.patch-applied: $(LIBGIT2_SRC_PATH)/sour
 	echo 1 > $@
 
 $(build_datarootdir)/julia/cert.pem:
```

Comment:
An issue that I just noticed with this while checking out these changes locally is that when the `MOZILLA_CACERT_VERSION` is changed the local file will not be updated.

One can achieve this by splitting this Make step into two steps one for downloading and checking the file and the other for installing it.

Reply:
Couldn't we remove the installed cert.pem during `make clean` (by adding `rm $(build_datarootdir)/julia/cert.pem`)?

Reply:
@vchuravy I noticed that the directory `$(build_datarootdir)/julia` still exists after `make clean`.